controller/projects.go

```diff
@@ -137,6 +145,6 @@ func (p *Projects) UpdateVersion(id uuid.UUID, version *semver.Version) error {
 }
 
 // Reset is not used in the API but for migration
-func (p *Projects) Reset(proj *model.Project) ([]*model.Account, error) {
+func (p *Projects) Reset(proj *model.Project) (*int, error) {
```

Comment:
I feel this should be refactored as mentioned before, it should just take the project ID and return the project model.

Reply:
Changed back to return new account models since I added accounts back anyways. I think it makes sense.